lessons/lesson1/concepts/how-do-i-decide-if-i-need-to-develop-for-virtual-reality-or-augmented-reality.md

```diff
@@ -6,3 +6,25 @@
 
 \*\*\*\*
 
+There are two main types of devices that deliver Windows Mixed Reality experiences:
+
+1. **Holographic devices**. These are characterized by the device's ability to place digital content in the real world as if it were really there.
+2. **Immersive devices**. These are characterized by the device's ability to create a sense of "presence" -- hiding the physical world, and replacing it with a digital experience.
+
+
+|Characteristic  |Holographic Devices  |Immersive Devices  |
+|---------|---------|---------|
+|**Example Device**     |    Microsoft HoloLens 2 </br> [insert picture]   |   Samsung HMD Odyssey+ </br> [insert picture]      |
+|**Display**     |    See-through display. Allows user to see the physical environment while wearing the headset.     |    Opaque display. Blocks out the physical environment while wearing the headset.     |
+|**Movement**     |    Full six-degrees-of-freedom movement, both rotation and translation.     |    Full six-degrees-of-freedom movement, both rotation and translation.     |
+
+
+Note, whether a device is connected to or tethered to a separate PC (via USB cable or Wi-Fi) or self-contained (untethered) does not reflect whether a device is holographic or immersive. Certainly, features that improve mobility lead to better experiences, and both holographic and immersive devices could be tethered or untethered.
+
+Technological advancement is what has enabled mixed reality experiences. There are no devices today that can run experiences across the entire spectrum. However, Windows 10 provides a common mixed reality platform for both device manufacturers and developers. Devices today can support a specific range within the mixed reality spectrum. Over time, new devices will expand that range. In the future, holographic devices will become more immersive, and immersive devices will become more holographic.
+
+Often, it is best to think what type of experience an application or game developer wants to create. The experiences will typically target a specific point or part on the spectrum. Then, developers should consider the capabilities of devices they want to target. For example, experiences that rely on the physical world will run best on HoloLens.
+
+- **Near physical reality**. Users remain present in their physical environment and are never made to believe they have left that environment.
+- **Fully mixed reality**. These experiences blend the real world and the digital world. Viewers who have seen the movie [Jumanji](https://en.wikipedia.org/wiki/Jumanji) can reconcile how the physical structure of the house where the story took place was blended with a jungle environment.
+- **Near digital reality**. Users experience a completely digital environment, and are unaware of what occurs in the physical environment around them.
```

Comment:
Please make the answer to which reality to choose more precisely and move the descriptions of the concepts into its own question.

Check out: What's the best reality for my app?

https://twitter.com/WindowsDocs/status/1212766878235602944